docs/data/stashcache/overview.md

```diff
@@ -0,0 +1,16 @@
+# Overview of StashCache
+
+StashCache is a data caching federation with many distributed caches spread across the U.S.
```

Comment:
This describes what StashCache is but doesn't really describe who the target audience is. Maybe word it something like this, expanding on `(for some value of large)` either here or below

> StashCache is a data caching federation used by large VOs (for some value of large) and the sites that support them.  The federation has many distributed caches spread across the U.S.

Reply:
Added comment about who uses StashCache.  It's actually the small VO's.  No big VO uses StashCache.

Reply:
Apart from the issue with the grammar in the sentence saying that stashcache is for researchers and sites isn't really informative: all of our external-facing software supports researchers or sites in some capacity.

If it's used by small VOs, what would prompt a small VO to set up stashcache origins/caches?